src/services/teams/events/user.js

```diff
@@ -30,19 +30,18 @@ const deleteUser = (app) => {
 			.then(() => {
 				const teamIds = teams.map((team) => team._id);
 				logger.info(
-					`Remove user ${userId} from the teams ${teamIds},`
-                    + 'if team.userIds empty then removed the team too.',
+					`Remove user ${userId} from the teams ${teamIds},` + 'if team.userIds empty then removed the team too.'
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6074601)